src/encoded/static/components/dataset.js

```diff
@@ -1279,6 +1303,18 @@ export class SeriesComponent extends React.Component {
         // Filter out any files we shouldn't see.
         const experimentList = context.related_datasets.filter(dataset => dataset.status !== 'revoked' && dataset.status !== 'replaced' && dataset.status !== 'deleted');
 
+        // If we display a table of related experiments, have to render the control to add all of
+        // them to the current cart.
+        let addAllToCartControl;
+        if (experimentList.length > 0) {
+            addAllToCartControl = (
+                <div className="experiment-table__header">
+                    <h4 className="experiment-table__title">{`Experiments in ${seriesTitle} ${context.accession}`}</h4>
+                    <CartAddAllElements elements={experimentList} css="experiment-table__control" />
```

Comment:
I wonder if you could add styles to the <CartAddAllElements> component without passing class names to the component. Maybe you could do something like .experiment-table__header { button { flex: 1 1 auto } }. I'm not sure if this will work for every case, or even if it's simpler, this is just a suggestion/comment.